cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -1043,7 +1052,7 @@ All supported versions of Cypher ran on the same Neo4j kernel.
 However, this changed in Neo4j 3.4 when the runtime was excluded from the compatibility layer.
 When you run, e.g. a `CYPHER 3.1` query in Neo4j 3.5, the query is planned with the 3.1 planner, but run with 3.5 runtime and kernel.
 The compatibility layer changed again in Neo4j 4.0 and it now includes only the Cypher language parser.
-When you run a `CYPHER 3.5` query, e.g., in Neo4j 4.3, Neo4j parses the older language features, but uses the 4.3 planner, runtime, and kernel to plan and run the query.
+When you run a `CYPHER 3.5` query, e.g., in Neo4j 4.4, Neo4j parses the older language features, but uses the 4.4 planner, runtime, and kernel to plan and run the query.
```

Comment:
If we want this to always be the most up to date version, we can use the attributes `{neo4j-version}` for _major.minor_, or `{neo4j-version-exact}` for _major.minor.patch_.

Reply:
I asked about that on Martins pr in this section, https://github.com/neo4j/neo4j-documentation/pull/1123#discussion_r643756982, and we could just have kept it as 4.3 here but I decided to update it when I anyway poked around in it (I guess it says something else in 4.2 anyway)